src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.html

```diff
@@ -169,3 +194,38 @@
     <ng-template ngPluralCase="other">{{ title }}s</ng-template>
   </span>
 </ng-template>
+
+<ng-template #dropdownTemplate>
+  <ng-container *ngFor="let data of dropdownData.total.category | keyvalue">
+    <li class="list-group-item">
+      <div class="d-flex pl-1 pb-2 pt-2">
+        <div class="ms-5 me-auto">
+          <span *ngIf="data.value.total"
+                [ngPlural]="data.value.total">
+              {{ data.value.total }}
+            <ng-template ngPluralCase="=0">{{ hwNames[data.key] }}</ng-template>
+            <ng-template ngPluralCase="=1">{{ hwNames[data.key] }}</ng-template>
+            <ng-template ngPluralCase="other">{{ hwNames[data.key] }}s</ng-template>
+          </span>
+        </div>
+        <span *ngIf="data.value.ok"
+              class="me-3">
+          <span>
+            {{ data.value.ok }}
+            <i class="text-success"
+               *ngIf="data.value.ok"
+               [ngClass]="[icons.success]">
+            </i>
+          </span>
+          <span *ngIf="data.value.error"
+                class="ms-2">
+                {{ data.value.error }}
+            <i class="text-danger"
+               [ngClass]="[icons.danger]">
+            </i>
+          </span>
+        </span>
+      </div>
+    </li>
+  </ng-container>
+</ng-template>
```

Comment:
i have made some changes locally which i'll just paste here since that'll be easier
```
diff --git a/src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.html b/src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.html
index 3a5ef55a14f..a193e2b6b62 100644
--- a/src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.html
+++ b/src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.html
@@ -1,6 +1,6 @@
 <hr>
 <li class="list-group-item">
-  <div class="d-flex pl-1 pb-2 pt-2">
+  <div class="d-flex pl-1 pb-2 pt-2 position-relative">
     <div class="ms-4 me-auto">
       <a [routerLink]="link"
          *ngIf="link && total > 0; else noLinkTitle"
@@ -12,7 +12,7 @@
         <ng-template ngPluralCase="other">{{ title }}s</ng-template>
       </a>
     </div>
-    <span class="me-3">
+    <span class="me-4">
       <ng-container [ngSwitch]="summaryType">
         <ng-container *ngSwitchCase="'iscsi'">
           <ng-container *ngTemplateOutlet="iscsiSummary"></ng-container>
@@ -28,8 +28,8 @@
         </ng-container>
       </ng-container>
     </span>
-    <span *ngIf="dropdownEnabled"
-          class="me-3">
+    <span *ngIf="dropdownData"
+          class="position-absolute end-0 me-2">
       <a (click)="toggleDropdown()"
          class="dropdown-toggle"
          [attr.aria-expanded]="dropdownToggled"
@@ -196,9 +196,10 @@
 </ng-template>
 
 <ng-template #dropdownTemplate>
-  <ng-container *ngFor="let data of dropdownData.total.category | keyvalue">
-    <li class="list-group-item">
-      <div class="d-flex pl-1 pb-2 pt-2">
+  <ng-container *ngFor="let data of dropdownData?.total.category | keyvalue">
+    <li class="list-group-item"
+        *ngIf="data?.value.error">
+      <div class="d-flex pb-2 pt-2">
         <div class="ms-5 me-auto">
           <span *ngIf="data.value.total"
                 [ngPlural]="data.value.total">
@@ -209,8 +210,7 @@
           </span>
         </div>
         <span *ngIf="data.value.ok"
-              class="me-3">
-          <span>
+              class="me-4">
             {{ data.value.ok }}
             <i class="text-success"
                *ngIf="data.value.ok"
@@ -218,13 +218,12 @@
             </i>
           </span>
           <span *ngIf="data.value.error"
-                class="ms-2">
+                class="me-4 ms-2">
                 {{ data.value.error }}
             <i class="text-danger"
                [ngClass]="[icons.danger]">
             </i>
           </span>
-        </span>
       </div>
     </li>
   </ng-container>
```

Reply:
this is what the outcome looks like. you can notice i removed all the items that are green and only shown the item which is in error.  I thought it'll be easier and makes more sense for a user rather than showing everything we have for hardware. Let me know your thought about it.

if we agree on the above then, we don't need to show the drop-down toggle and we can even by default toggle the dropdown when there is an error so user don't need to click to see it.

![image](https://github.com/ceph/ceph/assets/71764184/f710b6c4-6cf9-4b56-94be-e41b029c8a37)

Reply:
I don't mind one way or the other but,

on the demo we went the other way around, and neither in [patternfly](https://www.patternfly.org/patterns/dashboard/design-guidelines/) 
nor in [carbon](https://carbondesignsystem.com/patterns/status-indicator-pattern/) I see any reference to omitting the values when everything is fine, rather they use the green tick indicator as well.

Let me know what you think @nizamial09

Reply:
Leaving it as it is

Reply:
btw you can apply the html changes i made here, because it improves the alignment of the inventory card. Right now it looks a bit broken IMO, could you please apply them?

Reply:
addressed